db/seeds/categories/awrvi-2.0.yml

```diff
@@ -0,0 +1,354 @@
+        description: "10-49"
+      - index: 0.5
+        description: "50-99"
+      - index: 0.75
+        description: "100-499"
+      - index: 1.0
+        description: "> 500"
+    - name: Water source diversity
+      choices:
+      - index: 0
+        description: "1 surface or 1 ground"
+      - index: 0.25
+        description: "2 surface or 2 ground"
+      - index: 0.5
+        description: "1 surface and 1 ground"
+      - index: 0.75
+        description: "2 surface and 2 ground"
+      - index: 1.0
+        description: "> 2 of each"
+    - name: Drinking Water Treatment Technology (Boil, filter, chlorine, UV)
+      choices:
+      - index: 0
+        description: "None"
+      - index: 0.25
+        description: "1 of 4"
+      - index: 0.5
+        description: "2 of 4"
+      - index: 0.75
+        description: "3 of 4"
+      - index: 1.0
+        description: "4 of 4"
+    - name: Waste Water Treatment Technology
+      - choices:
+      - index: 0
+        description: "Honeybucket Lagoon"
+      - index: 0.25
+        description: "Outhouse"
+      - index: 0.5
+        description: "1˚ waste"
+      - index: 0.75
+        description: "2˚ waste"
+      - index: 1.0
+        description: "3˚ waste"
+    - name: Hydraulic gradient
+      choices:
+      - index: 0
+        description: "< -100"
+      - index: 0.25
+        description: "< -10"
+      - index: 0.5
+        description: "-10 to +0.9"
+      - index: 0.75
+        description: "+1.0 to +9.9"
+      - index: 1.0
+        description: "> +10"
+  - name: Physical - water quality
+    short_name: Water quality
+    categories:
+    - name: "Density of upstream development sites"
+      choices:
+      - index: 0
+        description: "> 0.1"
+      - index: 0.25
+        description: "0.08 - 0.009"
+      - index: 0.5
+        description: "0.05 - 0.079"
+      - index: 0.75
+        description: "0.02 - 0.049"
+      - index: 1.0
+        description: "0 - 0.019"
+    - name: "Streams with water quality data"
+      choices:
+      - index: 0
+        description: "None ever"
+      - index: 0.25
+        description: "Some WQ data > 10 years"
+      - index: 0.5
+        description: "Some WQ data < 10 years"
+      - index: 1.0
+        description: "Annual Monitoring"
+  - name: Physical - Permafrost Status
+    short_name: Permafrost Status
+    categories:
+    - name: PF distribution
+      choices:
+      - index: 0
+        description: "> 25% discontinous permafrost"
+      - index: 0.25
+        description: "1-25% discontinous permafrost"
+      - index: 0.5
+        description: "> 75% continous permafrost"
+      - index: 0.75
+        description: "1-75% continous permafrost and 1-75% no permafrost and 0% discontinous permafrost"
+      - index: 1.0
+        description: ">75% no permafrost"
+  - name: Physical - subsistence habitat
+    short_name: Subsistence habitat
+    categories:
+    - name: Anadromous Stream kilometers (%)
```

Comment:
Nope,  this is how it's defined in the datasheet I was given:
`% anadromous stream kilometers (from anadromous streams catalog)`